modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallCfgTest.java

```diff
@@ -105,6 +110,65 @@ public void testConfiguration() throws Exception {
 		Assert.assertEquals("testValue", dictionary.get(testKey));
 	}
 
+	@Test
+	public void testConfigurationDeprecatedFileExtension() throws Exception {
+		Boolean originalModuleFrameworkFileInstallCfgEnabled =
+			ReflectionTestUtil.getAndSetFieldValue(
+				PropsValues.class, "MODULE_FRAMEWORK_FILE_INSTALL_CFG_ENABLED",
+				false);
+
```

Comment:
Hey @kyle-miho,
Can you please move the code that get/restore the original value of `MODULE_FRAMEWORK_FILE_INSTALL_CFG_ENABLED` to the `setUp\tearDown` methods?

Thx